### DISCUSSION TITLE
Hybrid rendering.

### DISCUSSION THREAD

**Discussion by adityar15** [nuxt/nuxt#17943] [General]
Is there a way to use SSR on some pages and SSG on some pages in Nuxt 3? Something like Next getServerSideProps() and getStaticProps()

**Comment by TheAlexLichter** [comment]
Yes, this is planned! More details in https://github.com/nuxt/framework/discussions/560

**Reply by eat-sleep-code** [reply]
Is there an ETA on when a "Vercel ISR"-like revalidation feature will be available?   We have an upcoming project that would be dependent on such a feature.

**Reply by Rigo-m** [reply]
@eat-sleep-code I guess by Q2, since they have a nuxt 3 release candidate planned by the end of Q1

**Reply by BashCloud** [reply]
Hi @manniL 

I'm looking for ["Vercel ISR" with on-demand revalidation](https://vercel.com/docs/build-output-api/v3/features#on-demand-incremental-static-regeneration-isr) using Nuxt for multiple purposes. Is that available or is under development?

As per the information mentioned in docs: https://vercel.com/docs/concepts/incremental-static-regeneration/quickstart#on-demand-revalidation
<img width="812" alt="Screenshot 2023-07-29 at 6 20 14 PM" src="https://github.com/nuxt/nuxt/assets/16023060/0ed3b9d5-ecb5-47f2-97e1-abb2c293ee69">

If we can add support for on-demand revalidation with Nuxt, that will be really helpful.

**Reply by TheAlexLichter** [reply]
~~AFAIK this *should* work by now.~~ On demand is still not possible, but ISR in general yes.

**Reply by ldevoco** [reply]
Thanks for looking into this. Are there any plans to add on-demand soon? This is my most wanted feature.
